machine-learning/ml_tips/titanic.dig

```diff
@@ -120,25 +126,61 @@ _export:
     td>: queries/vectorize.sql
     source: titanic_norm_test
     create_table: test
+  +whole_dense:
+    td>: queries/vectorize_dense.sql
+    source: titanic_norm
+    create_table: whole_dense
+    feature_cardinality: ${td.last_results.max_categorical_cardinality} * 10
```

Comment:
Similarly to the above, we do need a comment what it does... 

It'd be more great if you have any external resources or examples that show x10 of max cardinality is sufficient for feature hashing.

Reply:
The number 10 is a heuristic number, but it is enough to limit the hashing space.

Reply:
Add note 997d035